Documentation/operations/upgrade.rst

```diff
@@ -318,6 +318,22 @@ Annotations:
   applications create new connections after the TTL specified by the upstream
   DNS server is expired.
 
+New Options
+~~~~~~~~~~~
+
+* ``routing-mode=native``: This option enables native-routing mode, in place of
+  ``tunnel=disabled``, now deprecated.
+* ``tunnel-protocol``: This option allows setting the tunneling protocol, in place
+  of e.g., ``tunnel=vxlan``.
+
+Deprecated Options
+~~~~~~~~~~~~~~~~~~
+
+* The ``tunnel`` option was deprecated and will be removed in v1.14. To enable
+  native-routing mode, you can set ``routing-mode=native`` (previously
+  ``tunnel=disabled``). To configure the tunneling protocol, you can set
+  ``tunnel-protocol=geneve`` (previously ``tunnel=geneve``).
```

Comment:
Should also mention `routing-mode=tunnel` here?
```suggestion
  ``tunnel=disabled``). To configure the tunneling protocol, you can set
  ``routing-mode=tunnel`` and ``tunnel-protocol=geneve`` (previously ``tunnel=geneve``).
```

Reply:
`routing-mode=tunnel` is already the default value (I think I said something different in a comment somewhere; it was one of Julian's comments).